pkg/acsengine/const.go

```diff
@@ -90,6 +102,12 @@ var KubeImages = map[api.OrchestratorVersion]map[string]string{
 		"dnsmasq":      "k8s-dns-dnsmasq-amd64:1.13.0",
 		"pause":        "pause-amd64:3.0",
 		"windowszip":   "v1.6.0intwinnat.zip",
+		"nodestatusfreq":   "10s",
+		"nodegraceperiod":   "40s",
+		"podeviction":   "5m0s",
+		"routeperiod":   "10s",
```

Comment:
I think the idea is that different versions of k8s will get different config values here. My latest commit, for example, only includes the `backoff*` and `ratelimit*` config properties in `1.6.6`. Does that address your question?